common/types/block.go

```diff
@@ -17,14 +17,19 @@ import (
 // BlockID is a 20-byte sha256 sum of the serialized block, used to identify it.
 type BlockID Hash20
 
-// String returns a short prefix of the hex representation of the ID.
+// String returns the hex representation of the ID.
 func (id BlockID) String() string {
-	return id.AsHash32().ShortString()
+	return id.AsHash20().String()
```

Comment:
Does this actually increase the length of the return value from this method? I think this was intentionally shortened (using ShortString) a few months ago to reduce the size of the logs

Reply:
This increases the length of the return value, but I changed all usages of this method to `ShortString()` to make it clear that they're getting the shortened version.